### ISSUE TITLE
People Picker dropdown list is not being announced in Android

### ISSUE THREAD

**Issue by yumikohey** [OfficeDev/office-ui-fabric-react#10004]
This is repro in Android and Win32 apps.

## Codepen demonstrating the issue
https://developer.microsoft.com/en-us/fabric#/controls/web/peoplepicker

## Actual behavior:
When users navigate with a physical keyboard up and down on the dropdown available list, voiceover should announce the name of the person.

## Expected behavior:
When users navigate with a physical keyboard up and down, voiceover is silent.

**Comment by marygans** [comment]
Hi @yumikohey thank you for submitting this issue! Can you specify the environment so I can try to repro?(e.g. Edge+narrator; Firefox + JAWS, etc.)
